admin/partials/settings/auto-update.php

```diff
@@ -0,0 +1,428 @@
+/**
+ * File: auto-update.php
+ *
+ * Show "Auto Update" section on settings page.
+ *
+ * @since      SINCEVERSION
+ * @package    Boldgrid_Backup
+ * @subpackage Boldgrid_Backup/admin/partials/settings
+ * @copyright  BoldGrid
+ * @version    $Id$
+ * @link       https://www.boldgrid.com
+ * @author     BoldGrid <support@boldgrid.com>
+ */
+
+// Get BoldGrid settings.
+\Boldgrid\Library\Util\Option::init();
+$boldgrid_backup_settings     = get_site_option( 'boldgrid_backup_settings', array() );
+$default_auto_update_settings = array(
+	'plugins' => array(),
+	'themes'  => array(),
+	'wpcore'  => array(),
+);
+$auto_update_settings         = isset( $boldgrid_backup_settings['auto_update'] ) ? $boldgrid_backup_settings['auto_update'] : $default_auto_update_settings;
+$translations                 = array(
+	'active'   => esc_attr__( 'Active', 'boldgrid-library' ),
+	'inactive' => esc_attr__( 'Inactive', 'boldgrid-library' ),
+	'parent'   => esc_attr__( 'Parent', 'boldgrid-library' ),
+);
+
+/**
+ * Get Heading markup.
+ *
+ * @since SINCEVERSION
+ *
+ * @param array $boldgrid_backup_settings Boldgrid Backup Settings.
+ * @param array $auto_update_settings Auto Update Settings from DB.
+ * @return string
+ */
+function get_heading_markup( $boldgrid_backup_settings, $auto_update_settings ) {
+	if ( empty( $auto_update_settings ) || 0 === $boldgrid_backup_settings['auto_backup'] ) {
+		$bbs_link_open  = '';
+		$bbs_link_close = '';
+
+		if ( empty( $_GET['page'] ) || 'boldgrid-backup-settings' !== $_GET['page'] ) { // phpcs:ignore WordPress.CSRF.NonceVerification.NoNonceVerification
+			$bbs_link_open  = '<a href="' . admin_url( 'admin.php?page=boldgrid-backup-settings&section=section_auto_rollback' ) . '">';
+			$bbs_link_close = '</a>';
+		}
+
+		return '
+			<div><p>' .
+			sprintf(
+				// translators: 1: HTML anchor open tag, 2: HTML anchor close tag, 3: HTML em open tag, 4: HTML em close tag, 5: Plugin Title.
+				esc_html__(
+					'You have %3$sAuto Backup Before Update%4$s disabled in the %1$s%5$s Backup and Restore Settings%2$s.  Please consider enabling the setting.',
+					'boldgrid-library'
+				),
+				$bbs_link_open,
+				$bbs_link_close,
+				'<em>',
+				'</em>',
+				'Total Upkeep'
+			) .
+			'</p></div>' . PHP_EOL;
+	}
+}
+
+/**
+ * Get Premium markup.
+ *
+ * @since SINCEVERSION
+ *
+ * @return string
+ */
+function get_premium_markup() {
+	$core        = apply_filters( 'boldgrid_backup_get_core', null );
+	$premium_url = $core->go_pro->get_premium_url( 'bgbkup-settings-auto-update' );
+
+	$premium_box = sprintf(
+		'
+		<div class="bg-box-bottom premium">
+			<p>
+				%1$s
+				%2$s
+			</p>
+		</div>',
+		/* 1 */ $core->go_pro->get_premium_button( $premium_url ),
+		/* 2 */ __( 'Upgrade to Premium for the option to configure a delay on updates!', 'boldgrid-backup' )
+	);
+
+	$premium_markup = '<div class="bg-box">
+		<div class="bg-box-top">' .
+			esc_html__( 'Configure When Auto Updates Occur', 'boldgrid-library' ) . '
```

Comment:
Was this file copied from the library? The text domain needs to be updated.

Reply:
Yes. It was moved from the library. corrected